xcm-support/src/lib.rs

```diff
@@ -12,10 +12,14 @@ use sp_std::{
 	result,
 };
 
-use xcm::v0::{Error, Junction, MultiAsset, MultiLocation, Result};
+use xcm::v0::{Error, Junction, MultiAsset, MultiLocation, Result, Xcm};
 use xcm_executor::traits::{FilterAssetLocation, LocationConversion, MatchesFungible, NativeAsset, TransactAsset};
 
-use frame_support::{log, traits::Get};
+use frame_support::{dispatch::DispatchResult, log, traits::Get};
+
+pub trait XcmHandler<AccountId> {
```

Comment:
shouldn't this be in cumulus / polkadot?

Reply:
There is no usage of this trait in Cumulus, not sure if it's good to add it there. We can probably open a PR to add it later.